CDCgov/prime-reportstream
# Added observation mapping table to local env

This PR adds the observation table to the local dev environment.

Test Steps:
1. Run the gradle task `reloadTables`
2. Run the prime CLI `prime lookuptables get -n observation-mapping -v 1 -o out.csv` to verify the table was created

## Changes
- Add `observation-mapping.csv` to `metadata/table/local`

## Checklist

### Testing

## Linked Issues
- Fixes #11948
